flink-ml-benchmark/src/test/java/org/apache/flink/ml/benchmark/BenchmarkTest.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.benchmark;
+
+import org.apache.flink.ml.benchmark.data.InputDataGenerator;
+import org.apache.flink.ml.benchmark.data.clustering.KMeansModelDataGenerator;
+import org.apache.flink.ml.benchmark.data.common.DenseVectorGenerator;
+import org.apache.flink.ml.clustering.kmeans.KMeans;
+import org.apache.flink.ml.clustering.kmeans.KMeansModel;
+import org.apache.flink.ml.common.distance.EuclideanDistanceMeasure;
+import org.apache.flink.ml.util.ReadWriteUtils;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+import org.apache.flink.table.api.bridge.java.StreamTableEnvironment;
+import org.apache.flink.test.util.AbstractTestBase;
+
+import org.apache.commons.io.FileUtils;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import java.io.File;
+import java.io.InputStream;
+import java.util.Map;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/** Tests benchmarks. */
+@SuppressWarnings("unchecked")
+public class BenchmarkTest extends AbstractTestBase {
+    @Rule public final TemporaryFolder tempFolder = new TemporaryFolder();
+
+    @Test
+    @SuppressWarnings({"unchecked"})
+    public void testParseJsonFile() throws Exception {
+        File configFile = new File(tempFolder.newFolder().getAbsolutePath() + "/test-conf.json");
+        InputStream inputStream =
+                this.getClass().getClassLoader().getResourceAsStream("benchmark-conf.json");
+        FileUtils.copyInputStreamToFile(inputStream, configFile);
+
+        Map<String, ?> benchmarks = BenchmarkUtils.parseJsonFile(configFile.getAbsolutePath());
+        assertEquals(benchmarks.size(), 1);
+        assertTrue(benchmarks.containsKey("KMeansModel-1"));
+
+        Map<String, ?> benchmark = (Map<String, ?>) benchmarks.get("KMeansModel-1");
+
+        KMeansModel stage =
+                ReadWriteUtils.instantiateWithParams((Map<String, ?>) benchmark.get("stage"));
```

Comment:
This logic could be covered by `testRunBenchmark()` if `testRunBenchmark()` uses `runBenchmark(StreamTableEnvironment tEnv, String name, Map<String, ?> params) `.

The test name `testParseJsonFile()` actually suggests this test just tests the `parseJsonFile()` method. 

And `ReadWriteUtils.instantiateWithParams()` is just an implementation detail of `runBenchmark()`.

Would it be simpler to just check that the output value of `parseJsonFile()` meets expectation?